### PR TITLE
Using protoc compiler instructions

### PR DESCRIPTION
**Pull request by sunithasomanna**
Added "Using protoc compiler" instructions in the Getting Started readme.

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

**Review by chauberahul1993 — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3690,4 +3691,98 @@ These checks run in parallel and take approximately 9 minutes to complete.
...
+# Using protoc compiler
+
+The protoc compiler provides a language-neutral, platform-neutral, extensible mechanism for defining communications protocols together with gRPC framework for communicating with Resource Aggregator for ODIM services. You can install the compiler, and create and compile your own protocols to the existing code.
+
+1. Install the protoc compiler by performing the following tasks:
+
+   1. Download the protoc compiler package:
+
+      ```
+   wget https://github.com/protocolbuffers/protobuf/releases/download/v3.19.1/protoc-3.19.1-linux-x86_64.zip.
```
Comment (shivacharanms) [inline README.md]: Remove the period at the end

Comment (shivacharanms) [reply README.md]: Change the command to 'wget https://github.com/protocolbuffers/protobuf/releases/download/v3.19.1/protoc-3.19.1-linux-x86_64.zip -P /tmp/'

File: README.md [hunk 2]
```diff
@@ -3690,4 +3691,98 @@ These checks run in parallel and take approximately 9 minutes to complete.
...
+      ```
+   
+   2. Create a directory `proto_files`:
```
Comment (shivacharanms) [inline README.md]: Remove step

File: README.md [hunk 3]
```diff
@@ -3690,4 +3691,98 @@ These checks run in parallel and take approximately 9 minutes to complete.
...
+   3. Install unzip (if not already installed):
+   
+      ```
+      sudo apt install unzip
+      ```
+   
+   4. Unzip the the protoc compiler package to the `proto_files` directory: 
+   
+      ```
+      protoc-3.19.1-linux-x86_64.zip -d proto_files`
```
Comment (shivacharanms) [inline README.md]: Update command to 'unzip /tmp/protoc-3.19.1-linux-x86_64.zip -d proto_files'

**Review by shivacharanms — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (sunithasomanna) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md]: done

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**